<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Clase_Estructuras</class>
 <widget class="QMainWindow" name="Clase_Estructuras">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>445</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>885</width>
    <height>445</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>885</width>
    <height>445</height>
   </size>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Clase</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(25, 34, 41);

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="algoritmosDatos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(33, 46, 54);
background-color: rgb(36, 49, 59);
border-top-color: rgb(42, 59, 71);
border-right-color: rgb(42, 59, 71);
border-buttom-color: rgb(42, 59, 71);
border-left-color: rgb(42, 59, 71);
color: rgb(16, 131, 214);

</string>
    </property>
    <property name="text">
     <string>Clase sobre Estructuras de Datos</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="iD">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="questionLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(33, 46, 54);
background-color: rgb(36, 49, 59);
border-top-color: rgb(42, 59, 71);
border-right-color: rgb(42, 59, 71);
border-buttom-color: rgb(42, 59, 71);
border-left-color: rgb(42, 59, 71);
color: rgb(16, 131, 214);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="imagen">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>411</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="texto">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>401</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top-color: rgb(42, 82, 158);
border-right-color:rgb(42, 82, 158);
border-buttom-color: rgb(42, 59, 71);
border-left-color: rgb(42, 82, 158);
color: rgb(200, 205, 192);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>370</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="1">
      <widget class="QPushButton" name="irQuiz1">
       <property name="styleSheet">
        <string notr="true">border-style: solid;
border-width: 1px;
;
background-color: rgb(0, 120, 100);
color: rgb(255, 255, 255);
border-radius: 5px;

</string>
       </property>
       <property name="text">
        <string>Quiz</string>
       </property>
       <property name="icon">
        <iconset resource="imagenes.qrc">
         <normaloff>:/images/images/quiz.png</normaloff>:/images/images/quiz.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="BotonSiguiente">
       <property name="styleSheet">
        <string notr="true">border-style: solid;
border-width: 1px;
;
background-color: rgb(129, 120, 255);
background-color: rgb(255, 176, 248);
color: rgb(255, 255, 255);
background-color: rgb(7, 99, 157);
border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>Siguiente</string>
       </property>
       <property name="icon">
        <iconset resource="imagenes.qrc">
         <normaloff>:/images/images/Siguiente.png</normaloff>:/images/images/Siguiente.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="BotonAnterior">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style: solid;
border-width: 1px;
;
background-color: rgb(129, 120, 255);
background-color: rgb(255, 176, 248);
color: rgb(255, 255, 255);
background-color: rgb(7, 99, 157);
border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>Anterior</string>
       </property>
       <property name="icon">
        <iconset resource="imagenes.qrc">
         <normaloff>:/images/images/Anterior.png</normaloff>:/images/images/Anterior.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="botonVolver">
       <property name="styleSheet">
        <string notr="true">border-style: solid;
border-width: 1px;
;
background-color: rgb(0, 120, 100);
color: rgb(255, 255, 255);
border-radius: 5px;

</string>
       </property>
       <property name="text">
        <string>Menu</string>
       </property>
       <property name="icon">
        <iconset resource="imagenes.qrc">
         <normaloff>:/images/images/Menu.png</normaloff>:/images/images/Menu.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
